pkg/kinflate/resource/appresource.go

```diff
@@ -21,8 +21,7 @@ import (
 	"k8s.io/kubectl/pkg/loader"
 )
 
-//  NewFromPath returns a Resource list given a resource path from manifest file.
-func NewFromPath(path string, loader loader.Loader) ([]*Resource, error) {
+func newFromPath(loader loader.Loader, path string) ([]*Resource, error) {
```

Comment:
Can we call this `resourcesFromPath` ? since this is internal it will not be invoked with `resource.` prefix so newFromPath reads very strange to me.